WasteWar/Assets/Scripts/Camera Controls/ZoomRouteDraw.cs

```diff
@@ -0,0 +1,22 @@
+﻿using UnityEngine;
+
+public class ZoomRouteDraw : MonoBehaviour
+{
+    [SerializeField]
+    private Transform[] controlPoints;
+    private Vector3 gizmosPosition;
+
+    private void OnDrawGizmos()
+    {
+        for (float t=0; t <= 1; t += 0.02f)
```

Comment:
pull numbers to consts with meaningful names

Reply:
this is a gizmo just for personal debugging, but i get it